LICENSE-3rdparty.csv

```diff
@@ -16,7 +16,6 @@ require,require-in-the-middle,MIT,Copyright 2016-2018 Thomas Watson Steen
 require,safe-buffer,MIT,Copyright Feross Aboukhadijeh
 require,shimmer,BSD-2-Clause,Copyright Forrest L Norvell
 require,url-parse,MIT,Copyright 2015 Unshift.io Arnout Kazemier the Contributors
-dev,@airbnb/node-memwatch,WTFPL,Copyright Lloyd Hilaiel
```

Comment:
Should we still have this since `node-memwatch` is still a dependency _somewhere_ in the project?

Reply:
For now we only include dependencies that are declared in the main `package.json`. Better handling of this along with automation will be introduced in a future PR.